### ISSUE TITLE
first select-image-rendering

### ISSUE THREAD

**Issue by prdt3e** [netresearch/t3x-rte_ckeditor_image#38]
There is a little bit of a logic error (in the JS) anyway
Currently it like this:
1. Select picture
2. Rendering
3. Dialog
4. Insert original image

the order should be like this:
1. Select picture
2. Get picture info (FileObject) (Ajax / SelectImageController.php ->getinfos)
3. Dialog with width, title, description
4. "again" ajax with rendering  (Ajax / SelectImageController.php -> processImage)  would then return the generated image.

As it is now, the original image is inserted the first time, only after saving the proccessed file appears.

**Comment by muh-nr** [comment]
@prdt3e 

Fixed it in branch issue38.
Please have a try.

**Comment by muh-nr** [comment]
Merged into master!

**Comment by prdt3e** [comment]
i like :)
